pages/_app.tsx

```diff
@@ -12,7 +12,6 @@ import { NEXT_SEO_DEFAULT } from "../next-seo.config";
 import "../styles.css";
 
 import AzureAppInsights from "../context/app-insight-client";
-const chatBaseBotId = process.env.NEXT_PUBLIC_CHATBASE_BOT_ID;
```

Comment:
do we need to remove this from the `.env.example` file and keeper?

Reply:
I removed NEXT_PUBLIC_ZENDESK_CHAT_KEY from `.env.example` and `NEXT_PUBLIC_CHATBASE_BOT_ID` from our `.github/.env`, we don't have these keys in keeper